api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -796,6 +796,189 @@
 			<column name="uuid" type="char(38)" />
 		</addColumn>
 	</changeSet>
+	<changeSet author="vshankar" id="201609171146-1">
+		<preConditions onFail="MARK_RAN">
+            <not><columnExists tableName="cohort_member" columnName="cohort_member_id"/></not>
+		</preConditions>
+		<dropForeignKeyConstraint constraintName="member_patient" baseTableName="cohort_member"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-1.1">
+		<preConditions onFail="MARK_RAN">
+			<not><columnExists tableName="cohort_member" columnName="cohort_member_id"/></not>
+		</preConditions>
+		<dropForeignKeyConstraint constraintName="parent_cohort" baseTableName="cohort_member"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-1.2">
+		<preConditions onFail="MARK_RAN">
+			<not><columnExists tableName="cohort_member" columnName="cohort_member_id"/></not>
+		</preConditions>
+		<comment>Dropping primary key for cohort_member</comment>
+		<dropPrimaryKey tableName="cohort_member"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-2">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="cohort_member" columnName="cohort_member_id"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="cohort_member_id" type="int">
+				<constraints nullable="false"/>
```

Comment:
Won't this fail for installations that have rows in this table that are upgrading? You need to add the nullable constraint after setting column values are set for existing rows. Also remember this column needs to allow autoIncrement since it is the primary key